WORKSPACE

```diff
@@ -94,22 +94,21 @@ load("@io_grpc_grpc_java//:repositories.bzl", "IO_GRPC_GRPC_JAVA_ARTIFACTS", "IO
 
 grpc_java_repositories()
 
-# Google Protocol Buffers.
-# https://github.com/protocolbuffers/protobuf
+### --- Base Proto Support --- ###
```

Comment:
So where is the `com_google_protobuf` archive installed now? Is it part of `rules_proto`? I was expecting we would move it up so we can control the exact version, instead of deleting it entirely.

Reply:
From Bazel docs I could find the suggested way of using proto rules is importing rules_proto, not protobuf sources. Do we care to pin protobuf sources or should we best let `rules_proto_dependencies()` do it for us?

Reply:
screen/596JVHnPJibmWgh

Reply:
Thanks, if `rules_proto` installs `com_google_protobuf` at a predictable version then that's fine.